src/services/facilityServices.js

```diff
@@ -59,6 +59,25 @@ const getFacilityByRegionId = async (id) => {
     return facilities;
 }
 
+const getFacilityByStateId = async (id) => {
+    let facilityList = await knex(constants.TABLE_FACILITY)
+    .select(constants.TABLE_REGION + '.' + constants.COLUMN_NAME + ' AS regionName', constants.TABLE_FACILITY + '.' + constants.COLUMN_NAME + ' AS facilityName', constants.COLUMN_IS_FUNCTIONAL)

```

Comment:
Could you put each selection on their own line for better reading?

Reply:
REALLY NICE IF WE CAN USE A LINTER FOR THIS DONT YOU TIHNK :)
Anyways, sure

Reply:
I'M WORKING ON IT AFTER I FINISH THE CURRENT STORY

Reply:
AHAHAHAHAHA